packages/table/src/headers/columnHeader.tsx

```diff
@@ -147,7 +162,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
 
         const rect = grid.getColumnRect(columnIndex);
         const handleSizeChanged = (size: number) => {
-            onResizeGuide([rect.left + size + 1]);
+            onResizeGuide([rect.left + size]);
```

Comment:
It was causing a mis-alignment between the resize guide and the reorder guide (reorder guide was centered on the line, resize guide was off by one). Things should still look identical given my new CSS changes!